content/_help/get-started/authentication-options._es.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Opciones de autenticacións
+permalink: /es/help/get-started/authentication-options/
```

Comment:
just to check my understanding, every page needs to define its permalink now, and include locale in it? If so, do we have any specs to help enforce that?

Reply:
And to this point, it looks like if someone were to create an article within Netlify CMS, it won't automatically populate a `permalink`, so it'd require someone to manually edit it later in code.

Reply:
I wonder if this is something where we could create a Jekyll plugin to dynamically set the permalink using the content's `lang` value.

Reply:
Looking at it. I will create a separate branch to address this. I think we can simplify the collection path given in _config.yaml but will require a change to how we use that locale_url_filter as well. Will address that in a followup Ticket.

Reply:
Created ticket LG-5466 to address this.